src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -78,10 +78,14 @@
         programGroup = Metrics.class
 )
 public class CollectAlignmentSummaryMetrics extends SinglePassSamProgram {
-    static final String USAGE_SUMMARY = "Produce a summary of alignment metrics from a SAM or BAM file";
-    static final String USAGE_DETAILS = "Using read outputs from high throughput sequencing (HTS) technologies, this tool provides " +
-            "metrics regarding the quality of read alignments to a reference sequence, as well as the proportion of the reads " +
-            "that passed machine signal-to-noise threshold quality filters (Illumina)."+
+    static final String USAGE_SUMMARY = "Produces a summary of alignment metrics from a SAM or BAM file";
+    static final String USAGE_DETAILS = "This tool takes an aligned SAM/BAM file, and produces metrics detailing the quality of the read " +
+            "alignments as well as the machine-generated base quality scores within these reads.  " +
+            "" +
+            "For Additional information about Illumina's base quality" +
```

Comment:
Unwanted capital A (For Add..)

By the way, I don't think "the machine-generated base quality scores within these reads" is equivalent to the original text "proportion of the reads that passed machine signal-to-noise threshold quality filters (Illumina)". This rewrite changes the meaning conveyed here; I'm not sure this is correct.